bundles/org.openhab.binding.chatgpt/src/main/resources/OH-INF/i18n/chatgpt.properties

```diff
@@ -12,6 +12,12 @@ thing-type.chatgpt.account.description = Account at OpenAI that is used for acce
 
 thing-type.config.chatgpt.account.apiKey.label = API Key
 thing-type.config.chatgpt.account.apiKey.description = API key to access the account
+thing-type.config.chatgpt.account.apiUrl.label = API URL
+thing-type.config.chatgpt.account.apiUrl.description = The server API where to reach the AI service.
+thing-type.config.chatgpt.account.apiUrl.option.https\://api.openai.com/v1/chat/completions = https://api.openai.com/v1/chat/completions
```

Comment:
I have a big doubt. Do you think this entry is valid ?

Reply:
Maybe you should just omit the entry, translating a URL has no sense by the way.

Same for model URL.

Reply:
Well, this file is being generated, so even if I manually remove it here, it will reappear on the next generation.
I would assume that the entry is correct - if not, we should rather adapt the i18n generation.

Reply:
The i18n generator is an helper but we have several exceptions. I believe we have to avoid adding translation work in Crowdin when it is useless.

Reply:
Ok, I have removed the two lines.